Sources/ConcurrencyExtras/AsyncStream.swift

```diff
@@ -79,7 +79,7 @@ extension AsyncStream {
 extension AsyncSequence {
   /// Erases this async sequence to an async stream that produces elements till this sequence
   /// terminates (or fails).
-  public func eraseToStream() -> AsyncStream<Element> {
+  public func eraseToStream() -> AsyncStream<Element> where Self: Sendable {
```

Comment:
I understand this, but unfortunately this means that because `Combine` is stuck in the past, you can't do this anymore:
```swift
publisher.values.eraseToStream()
```

Reply:
Does a `@preconcurrency import` help with that? Or worst case, a `@retroactive @unchecked Sendable` conformance for the publisher in question?

Reply:
> Does a @preconcurrency import help with that?

Nope

> a @retroactive @unchecked Sendable conformance for the publisher in question?

Yeah but that would silence all other issues related to mis-using `Publisher`s.

Reply:
Any chance of relaxing this requirement? We'll be stuck on 1.1.0 otherwise to be able to keep supporting Combine.

Reply:
@NachoSoto We recently introduced a conditional conformance to `UncheckedSendable` here: https://github.com/pointfreeco/swift-concurrency-extras/pull/44

Can you use that?

Reply:
Yes that's perfect thank you!

Ended up doing this (`AnyAsyncSequence` is our own type to deal with the `Failure` `associatedtype` availability):
```swift
extension Publisher where Output: Sendable {
  public func eraseToAnyAsyncSequence() -> AnyAsyncSequence<Output> {
    // Combine does not support Swift concurrency, so we rely on `ConcurrencyExtras`
    // allowing this.
    UncheckedSendable(values).eraseToAnyAsyncSequence()
  }
}
```